EIPS/draft-eip-multi-algorithm-pki.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Support for transactions signed with non-secp256k1 algorithms
+description: This EIP adds a standardized way to support additional algorithms to signed transactions with
+author: James Kempton (@SirSpudlington)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-04-12
+requires: 2718, 2930, 1559, 4844
+---
+
+## Abstract
+
+This EIP introduces a new wrapper type transaction to allow for alternative signature algorithms to sign off on an ethereum transactions.
+
+## Motivation
+
+As quantum computers are getting more advanced and several new post-quantum (PQ) algorithms have been designed. These algorithms all contain drawbacks such as large key sizes (>1KiB), large signature sizes or long verification times. These issues make them more expensive to compute and store than the current secp256k1 curve in use (as of 2025-04-12).
+
+This EIP provides a future-proofed solution to the several algorithms by adding a standardized way to represent alternative algorithms within a transaction.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Parameters
+
+| Constant | Value |
+| - | - |
+| `ALG_TX_TYPE` | `Bytes1(0x04)` |
```

Comment:
This is the same as an EIP 7702 transaction? I suggest using `0x07`: `0x05` is used as magic in authority lists (by EIP 7702) and `0x06` is currently reserved by EOFs TXCREATE tx